system/Database/Forge.php

```diff
@@ -1041,9 +1096,28 @@ protected function _processIndexes(string $table)
         return $sqls;
     }
 
```

Comment:
Added second parameter $asQuery. This tells the function to return a stand alone query rather than the partial sql used when creating a table.

Reply:
Note that instead of returning a string the method now returns an array